datashuttle/datashuttle.py

```diff
@@ -183,6 +190,22 @@ def download_data(
             "download", sub_names, ses_names, data_type, dry_run
         )
 
+    def upload_all(self):
+        """
+        Convenience function to upload all data.
+        Alias for:
+            project.upload_data("all", "all", "all")
+        """
+        self.upload_data("all", "all", "all")
+
+    def download_all(self):
+        """
+        Convenience function to download all data.
+        Alias for:
+            project.download_data("all", "all", "all")
+        """
+        self.download_data("all", "all", "all")
```

Comment:
maybe we can have "all" "all" "all" to be more explicit in the future

Reply:
do you mean to change as below in the docstring?
```
        Alias for:
            project.download_data(sub_names="all",
                                               ses_names="all", 
                                               data_type="all")
```

Reply:
It confuses me to pass the concept of the action of "downloading all" with a string. What else can you pass?
I see that then you use it in another method with an if statement `if ses_names not in ["all", ["all"]]:`. 
I suggest to refactor in the future with `download_all_subnames=True` etc.. in case there is the chance of this method to be part of the user facing API

Reply:
I see this is a good point, currently you can pass different list / string e.g. `"sub-001"`, `["sub-001",, "sub_002", "all_non_ses"]` for various combinations of files, for the user this is not super-clear. In future hopefully 99% of use will be through TUI but I'll add a note on the relevant issue thanks